lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -39,6 +39,64 @@ def close(message: nil)
       load_attributes_from_response(post(:close, {}, body.to_json))
     end
 
+    def request_fulfillment(fulfillment_order_line_items:, message: nil)
```

Comment:
> The fulfillment order line items to be requested for fulfillment. If left blank, all line items of the fulfillment order are requested for fulfillment.
https://help.shopify.com/en/api/reference/shipping-and-fulfillment/fulfillmentrequest#create-2020-01

It seems like `fulfillment_order_line_items` can also be left blank